TimedArithmeticQuiz/src/components/QuizComplete.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types"
+import { ResponseList } from "./ResponseList"
+import { QuizHeader } from "./QuizHeader"
+
+export const QuizComplete = ({correctAnswerQuestions, wrongAnswerQuestions, score}) => {
+  return <div>
+    <QuizHeader score={score} />
+    <ResponseList correctAnswers={true} list={correctAnswerQuestions} />
+    <ResponseList correctAnswers={false} list={wrongAnswerQuestions} />
+  </div>
+}
+
+QuizComplete.propTypes = {
+  correctAnswerQuestions: PropTypes.array,
```

Comment:
We should also specify the type of elements expected in the array.

Reply:
done